qp_shogun/sortmerna/__init__.py

```diff
@@ -23,6 +23,17 @@
 outputs = {'Non-ribosomal reads': 'per_sample_FASTQ',
            'Ribosomal reads': 'per_sample_FASTQ'}
 
+# defining default parameter set AKA what's going to be shown to the user
+# as options for the command
+dflt_param_set = {
+    'Ribosomal read filtering': {
+        'Output blast format': 1,
```

Comment:
Quick question, the type of the arguments here is expressed as an integer, and as a string in the `opt_params` declaration. Is that right?

Reply:
As far as I remember, that is correct.